pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -86,41 +99,42 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 
 	wxBoxSizer& s_customsize( *new wxBoxSizer( wxHORIZONTAL ) );
 	s_customsize += m_text_WindowWidth;
-	s_customsize += Label( L"x" )                     | StdExpand();
+	s_customsize += Label( L"x" ) | StdExpand();
 	s_customsize += m_text_WindowHeight;
 
 	wxFlexGridSizer& s_AspectRatio( *new wxFlexGridSizer( 2, StdPadding, StdPadding ) );
 	//s_AspectRatio.AddGrowableCol( 0 );
 	s_AspectRatio.AddGrowableCol( 1 );
 
-	s_AspectRatio += Label(_("Aspect Ratio:"))        | pxMiddle;
-	s_AspectRatio += m_combo_AspectRatio              | pxAlignRight;
-	s_AspectRatio += Label(_("Custom Window Size:"))  | pxMiddle;
-	s_AspectRatio += s_customsize                     | pxAlignRight;
+	s_AspectRatio += Label(_("Aspect Ratio:")) | pxMiddle;
+	s_AspectRatio += m_combo_AspectRatio | pxAlignRight;
+	s_AspectRatio += Label(_("FMV Aspect Ratio switch:")) | pxMiddle;
```

Comment:
I'm actually gonna rename the label to **FMV Aspect Ratio Override:**. It's been bugging me.